samples/ingredients/disks/create_kms_encrypted_disk.py

```diff
@@ -0,0 +1,70 @@
+#  Copyright 2022 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# This is an ingredient file. It is not meant to be run directly. Check the samples/snippets
+# folder for complete code samples that are ready to be used.
+# Disabling flake8 for the ingredients file, as it would fail F821 - undefined name check.
+# flake8: noqa
+from typing import Optional
+
+from google.cloud import compute_v1
+
+
+# <INGREDIENT create_kms_encrypted_disk>
+def create_kms_encrypted_disk(project_id: str, zone: str, disk_name: str, disk_type: str,
+                              disk_size_gb: int, kms_key_name: str,
+                              disk_link: Optional[str] = None, image_link: Optional[str] = None) -> compute_v1.Disk:
+    """
+    Creates a new disk in a project in given zone. If disk_link and image_link are not provided, the disk will be
```

Comment:
Creates a zonal disk in a project. If you do not provide values for `disk_link` or `image_link`, an empty disk will be created.

Maciej, isn't it either source disk or source image? Hence suggested 'or' in the above comment.

Reply:
Yes, those two are mutually exclusive. `or` fits better here.